catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -258,6 +243,96 @@ public void testMetacardTypeValuesCacheEnabled() throws Exception {
         assertReplies(replies);
     }
 
+    @Test
+    public void testReducedMaxQueryResults() throws Exception {
+
+        framework = createFramework(20);
```

Comment:
✏️  You could make 20 and 7 final variables here and use them in the setup and in the asserts to make it clore clear why you're expecting 20L and 7.